packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -354,7 +387,7 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
    * Note: This won't work in case of multiple legends selection.
    */
   private _getHighlightedLegend() {
-    return this.state.selectedLegend || this.state.activeLegend;
+    return this.state.selectedLegends.length > 0 ? this.state.selectedLegends : [this.state.activeLegend];
```

Comment:
>return this.state.selectedLegends.length > 0 ? this.state.selectedLegends : [this.state.activeLegend]; [](http://example.com/codeflow?start=4&length=102)

this wont work if 2 legend are selected and a third legend is hovered

Reply:
But, in line multiple chart, on multiple legend selection, legend hover is disabled. Shall the experience be not the same?